lib/repository/models/chat.dart

```diff
@@ -835,6 +835,11 @@ class Chat {
         [this.guid]);
   }
 
+  Future<Rx<Message>> get latestMessage async {
+    List<Message> latests = await Chat.getMessages(this, limit: 1);
+    return latests.first.obs;
```

Comment:
This won't do anything meaningful because you don't ever set latest.first to a different value. I think it will work the same way if you just get rid of the reactive state here entirely (no .obs and no Rx).

Reply:
Ah ok good to know. I was just trying some stuff because the speech bubble was occasionally not showing up on a new message. One of the changes in the latest commit fixed it but ig not this one lol.